bagelbids/tests/test_utility.py

```diff
@@ -0,0 +1,185 @@
+import pandas as pd
+import pytest
+from pydantic.main import ModelMetaclass
+
+from bagelbids import mappings, models
+from bagelbids.cli import (
+    are_not_missing,
+    get_columns_about,
+    get_transformed_values,
+    is_missing_value,
+    map_categories_to_columns,
+    map_tools_to_columns,
+    transform_age,
+)
+
+
+@pytest.fixture
+def generate_context():
```

Comment:
sorry for not being clear here. I thought you could just do:

```python
from bagelbids.cli import gen_context

@pytest.fixture
def get_test_context():
  return gen_context()
```
And then use that as an alternative to calling `gen_context` directly repeatedly (because it always returns the same thing anyway). It's not a good idea to copy the function like this because now we don't test the `bagelbids.cli.gen_context` function any more.

Reply:
AH I understand, that does make more sense 😅 good catch